autorandr.py

```diff
@@ -780,8 +780,8 @@ def apply_configuration(new_configuration, current_configuration, dry_run=False)
 
     # Enable the remaining outputs in pairs of two operations
     operations = disable_outputs + enable_outputs
-    for index in range(0, len(operations), 2):
-        argv = base_argv + list(chain.from_iterable(operations[index:index + 2]))
+    for op in operations:
+        argv = base_argv + list(op)
```

Comment:
Going one by one is an issue if there's still an output to disable, because then the first call would disable the last output. You'll have to add code like

    if disable_outputs:
        do a single call that disables the remaining outputs and enables the first output

and then handles the remaining ones one by one.

Reply:
I understand what you're saying, but I don't see the problem. XRandR works fine if there's not a single output active. So even if all outputs get disabled, the fact that there are calls queued that will enable them again, should make this a safe operation, no?